### PR TITLE
[mage-1-11-1 < E-graphQL] Update docs after refactor for GraphQL and fix some mistakes

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

The description of algorithms that are util + if they were listed in the wrong place on the available algorithms page. 
The changes in parameters and outputs due to GraphQL.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
https://github.com/memgraph/mage/pull/358

Closes:
(paste the link to the issue it closes)

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/advanced-algorithms/available-algorithms/collections.mdx [hunk 1]
```diff
@@ -22,29 +22,35 @@ sorting, and modification for efficient data handling.
...
 
 | Trait               | Value                                                 |
-| ------------------- | ----------------------------------------------------- |
-| **Module type**     | <Highlight color="#FB6E00">**util**</Highlight>  |
-| **Implementation**  | <Highlight color="#FB6E00">**C++**</Highlight>        |
-| **Graph direction** | <Highlight color="#FB6E00">**directed**</Highlight> |
-| **Edge weights**    | <Highlight color="#FB6E00">**weighted**</Highlight> |
-| **Parallelism**     | <Highlight color="#FB6E00">**sequential**</Highlight> |
+| ------------------- | ---------- |
+| **Module type**     | algorithm  |
```
Comment (mpintaric55334) [inline pages/advanced-algorithms/available-algorithms/collections.mdx]: This should be util

Comment (mpintaric55334) [inline pages/advanced-algorithms/available-algorithms/collections.mdx]: Should be util

File: pages/advanced-algorithms/available-algorithms/collections.mdx [hunk 2]
```diff
@@ -29,75 +32,66 @@ sorting, and modification for efficient data handling.
...
-## Procedures
+## Functions
 
 If a function requires elements to be comberable, `Path`, `List` and `Map` are
 uncomparable. <br />
 Only `Numeric` data types can be used in `sum()` and `avg()` functions, so only
 `Int` and `Double` data types are allowed.
 
-### `remove_all()`
+### `sort()`
```
Comment (mpintaric55334) [inline pages/advanced-algorithms/available-algorithms/collections.mdx:42]: In the old docs, we wrote the header with function(arguments), I see that you changed it to just function(), is this something we should abide in the future?

File: pages/advanced-algorithms/available-algorithms/collections.mdx (hunk 1 above)
Comment (vpavicic) [reply pages/advanced-algorithms/available-algorithms/collections.mdx]: I will change it tomorrow

File: pages/advanced-algorithms/available-algorithms/collections.mdx (hunk 2 above)
Comment (vpavicic) [reply pages/advanced-algorithms/available-algorithms/collections.mdx:42]: Yup... so please check these first couple of algorithms -> those are the ones i managed to polish... each weak i will polish 5-6 algorithms and the idea is to have them all look alike... so these are kind of the templates i'll be changing all the rest of them to

Comment (mpintaric55334) [reply pages/advanced-algorithms/available-algorithms/collections.mdx:42]: Looks good!

**Comment by vpavicic** [comment]
@mpintaric55334 can I merge into the mage-1-11-1 branch?

**Pull request merged**